app/components/task_list_items/confirm_site_notice_component.rb

```diff
@@ -16,21 +16,21 @@ def link_text
     end
 
     def link_path
-      if @site_notice.displayed_at.nil?
-        edit_planning_application_site_notice_path(planning_application, site_notice)
-      else
+      if @site_notice.document.present?
         planning_application_site_notice_path(planning_application, site_notice)
+      else
+        edit_planning_application_site_notice_path(planning_application, site_notice)
       end
     end
 
     def status_tag_component
-      if @site_notice.displayed_at.nil?
+      if @site_notice.document.present?
         StatusTags::BaseComponent.new(
-          status: "not_started"
+          status: "complete"
```

Comment:
Perhaps

```suggestion
          status: @site_notice.document.present? ? "complete" : "not_started"
```

instead of a conditional?